res/controllers/Numark-Mixtrack-3-scripts.js

```diff
@@ -1085,14 +1052,28 @@ NumarkMixtrack3.init = function(id, debug) {
     print("");
 
     // Set soft-takeover for all Sampler volumes
-    for (i = engine.getValue("[Master]", "num_samplers"); i >= 1; i--) {
+    for (var i = engine.getValue("[Master]", "num_samplers"); i >= 1; i--) {
         engine.softTakeover("[Sampler" + i + "]", "pregain", true);
     }
 
-    for (i = engine.getValue("[Master]", "num_decks"); i >= 1; i--) {
+    for (var i = engine.getValue("[Master]", "num_decks"); i >= 1; i--) {
```

Comment:
Okay, just pointing it out because many people who contribute mapping scripts aren't that familiar with JS.